File: exercicios/ascencio/cap5/cpp/EX19_B.cpp
#include <stdio.h>
int main()
{ int m, n, soma, i;
  do
  {
  printf("\nDigite um valor para m \n");
  scanf("%d%*c",&m);
  printf("Digite um valor para n \n");
  scanf("%d%*c",&n);
  if (m < n)
    { soma = 0;
      for (i=m;i<=n;i++)
		soma = soma + i;
      printf("Soma dos inteiros entre %d e %d = %d",m,n,soma);
    }
}  
  while ( m < n);  
  return 0;	
}

File: exercicios/ascencio/cap5/java/EX19_B.java
import java.util.*;
public class EX19_B
{
  public static void main (String args[])
  	{
  		Scanner dado = new Scanner(System.in);
  		int m, n, soma, i;
  		do
		 { 
		 	System.out.println("Digite um valor para m ");
  			m = dado.nextInt();
  			System.out.println("Digite um valor para n ");
  			n = dado.nextInt();
		 	if (m < n)
		 	{
		   	soma = 0;
      		for (i=m;i<=n;i++)
				soma = soma + i;
			System.out.println("Soma dos inteiros entre "+m+" e "+n+" = "+soma);
			}
		 }	
		 while (m < n);
  	}
}